### PR TITLE
Add converters for timestamps and memsize

### PR DESCRIPTION
**Pull request by MichaelDeciso**
**Do not merge this yet. I'll explain below.**

Opening at the request of @AdSchellevis .

This adds more converters to Bootgrid, so that it can sort correctly on memsize, for example.

With the functions in question being generic, I think they would be better at home in a shared JS library, so that we can call them from other places. Of course, we would still need to define them here, but they would be wrappers to real ones. The same applies to the memsize prefixes (K/M/G/...).

### CODE REVIEW

File: src/public.js [hunk 1]
```diff
@@ -249,6 +249,36 @@ Grid.defaults = {
             // default converter
             from: function (value) { return value; },
             to: function (value) { return value; }
+        },
+        timestamp: {
+            to: function(value) {
+                return (new Date((+value) * 1000)).toLocaleString(navigator.language, { timeZone: 'UTC' });
```
Comment (fabianfrz) [inline src/public.js:255]: maybe use momentjs

File: src/public.js [hunk 2]
```diff
@@ -249,6 +249,36 @@ Grid.defaults = {
...
+                for (let exponent = modifiers.length - 1; exponent >= 0; exponent--) {
+                    if (modifier === modifiers[exponent]) {
+                        ret *= Math.pow(1024, exponent);
+                        break;
+                    }
+                }
+                return ret;
+            },
+            to: function (value) {
+                const modifiers = ['', 'K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'];
```
Comment (fabianfrz) [inline src/public.js:273]: this are KiB, MiB, …

Comment (MichaelDeciso) [reply src/public.js:273]: No, these are the suffixes used in the SIZE and RES columns on `https://ip.of.opnsense/ui/diagnostics/activity`.

File: src/public.js (hunk 1 above)
Comment (MichaelDeciso) [reply src/public.js:255]: For one line? That seems like overkill to me. (Incidentally, I have taken this code from a PR you created against the `core` repo.)

**Comment by fabianfrz** [comment]
@MichaelDeciso this is my boolean helper for example:

https://github.com/opnsense/plugins/blob/master/net/frr/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsospf.volt#L414-L423

File: src/public.js (hunk 1 above)
Comment (fabianfrz) [reply src/public.js:255]: Ok, maybe I've seen it too late too ;)

It is in core anyway: https://github.com/opnsense/core/blob/master/src/opnsense/www/js/moment-with-locales.min.js

**Pull request merged**